From 3e8ff68502bf5de333db7213d9e27e0b9e8cc36e Mon Sep 17 00:00:00 2001
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Mon, 29 Aug 2016 12:34:42 -0400
Subject: [PATCH 7/7] avoid regenerating defsincdate (use shipped file)

upstream ships doc/defsincdate in its tarballs.  but doc/Makefile.am
tries to rewrite doc/defsincdate if it notices that any of the files
have been modified more recently, and it does so assuming that we're
running from a git repo.

However, we'd rather ship the documents cleanly without regenerating
defsincdate -- we don't have a git repo available (debian builds from
upstream tarballs) and any changes to the texinfo files (e.g. from
debian/patches/) might result in different dates on the files than we
expect after they're applied by dpkg or quilt or whatever, which makes
the datestamp unreproducible.
---
 doc/Makefile.am | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index d47d83ede..c0a81b0b9 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -177,15 +177,6 @@
 
 dist-hook: defsincdate
 
-defsincdate: $(gnupg_TEXINFOS)
-	: >defsincdate ; \
-	if test -e $(top_srcdir)/.git; then \
-	  (cd $(srcdir) && git log -1 --format='%ct' \
-               -- $(gnupg_TEXINFOS) 2>/dev/null) >>defsincdate; \
-        elif test x"$SOURCE_DATE_EPOCH" != x; then   \
-	   echo "$SOURCE_DATE_EPOCH" >>defsincdate ; \
-	fi
-
 defs.inc : defsincdate Makefile mkdefsinc
 	incd="`test -f defsincdate || echo '$(srcdir)/'`defsincdate"; \
 	./mkdefsinc -C $(srcdir) --date "`cat $$incd 2>/dev/null`" \
-- 
2.27.0

